<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1483</width>
    <height>1009</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HA525</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/resources.qrc">
    <normaloff>:/icon/icon.png</normaloff>:/icon/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1Name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>435</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Bird Eye View</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>425</y>
      <width>480</width>
      <height>270</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-color: rgb(186, 189, 182);
border-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(173, 40, 40);</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2Name">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>435</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Color Filtered</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>425</y>
      <width>480</width>
      <height>270</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-color: rgb(186, 189, 182);
border-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label4Name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>715</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Filtered</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label3Name">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>435</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Scharr Filtered</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>705</y>
      <width>480</width>
      <height>270</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-color: rgb(186, 189, 182);
border-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>425</y>
      <width>480</width>
      <height>270</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-color: rgb(186, 189, 182);
border-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label5Name">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>715</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Detected</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label6">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>705</y>
      <width>480</width>
      <height>270</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-color: rgb(186, 189, 182);
border-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label6Name">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>715</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Ground Level View</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label5">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>705</y>
      <width>480</width>
      <height>270</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-color: rgb(186, 189, 182);
border-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>981</width>
      <height>40</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #eee, stop:1 #ccc);
border: 1px solid #777;
width: 13px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 4px;
}

QSlider::handle:horizontal:hover {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #fff, stop:1 #ddd);
border: 1px solid #444;
border-radius: 4px;
}

QSlider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #999;
}</string>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>55</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-color: rgb(186, 189, 182);
border-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label0Name">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>65</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Input</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label7Name">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>65</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label7">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>55</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 1px;
border-color: rgb(186, 189, 182);
border-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label0FPS">
    <property name="geometry">
     <rect>
      <x>639</x>
      <y>385</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>FPS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label7FPS">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>385</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>FPS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label3FPS">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>665</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>FPS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label6FPS">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>945</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>FPS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label2FPS">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>665</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>FPS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label5FPS">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>945</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>FPS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label1FPS">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>665</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>FPS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label4FPS">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>945</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>FPS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>10</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>10</y>
      <width>20</width>
      <height>40</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QRadioButton::indicator:unchecked{
    color: #ffffff;
    background-color: #ffffff;
    border: 1px solid #ffffff;
	border-radius: 4px
}

QRadioButton::indicator:checked {
    background-color: #5280ff;
    border: 2px solid #ffffff;
border-radius: 4px
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>with CUDA</string>
    </property>
   </widget>
   <zorder>label7</zorder>
   <zorder>label5</zorder>
   <zorder>label4</zorder>
   <zorder>label3</zorder>
   <zorder>label2</zorder>
   <zorder>label1</zorder>
   <zorder>startButton</zorder>
   <zorder>label1Name</zorder>
   <zorder>stopButton</zorder>
   <zorder>label2Name</zorder>
   <zorder>label4Name</zorder>
   <zorder>label3Name</zorder>
   <zorder>label5Name</zorder>
   <zorder>label6</zorder>
   <zorder>label6Name</zorder>
   <zorder>openButton</zorder>
   <zorder>connectButton</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>label0</zorder>
   <zorder>label0Name</zorder>
   <zorder>label7Name</zorder>
   <zorder>label0FPS</zorder>
   <zorder>label7FPS</zorder>
   <zorder>label3FPS</zorder>
   <zorder>label6FPS</zorder>
   <zorder>label2FPS</zorder>
   <zorder>label5FPS</zorder>
   <zorder>label1FPS</zorder>
   <zorder>label4FPS</zorder>
   <zorder>labelTime</zorder>
   <zorder>radioButton</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1483</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(226, 230, 235);
background-color: rgb(0, 0, 0);</string>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_HA525"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout_HA525">
   <property name="text">
    <string>About HA525</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resource/resources.qrc"/>
 </resources>
 <connections/>
</ui>
